mix.exs

```diff
@@ -32,7 +32,8 @@ defmodule Scidata.MixProject do
     [
       {:ex_doc, ">= 0.24.0", only: :dev, runtime: false},
       {:nimble_csv, "~> 1.1"},
-      {:jason, "~> 1.0"}
+      {:jason, "~> 1.0"},
+      {:img_decode, "~> 0.1.0", github: "cocoa-xu/img_decode"}
```

Comment:
Final suggestion. Let's make this an optional dependency:

```suggestion
      {:img_decode, "~> 0.1.0", github: "cocoa-xu/img_decode", optional: true}
```

Then at the beginning of the download function you check this:

```elixir
unless Code.ensure_loaded?(ImgDecode) do
  raise "ImgDecode is missing, please add `{:img_decode, "~> 0.1"}` as a dependency to your mix.exs"
end
```